antibodies/pixel_analysis_workflow1.py

```diff
@@ -9,6 +9,7 @@
 from batchlib import run_workflow
 from batchlib.preprocessing import Preprocess
 from batchlib.segmentation import IlastikPrediction
+from batchlib.analysis.pixel_level_analysis import PixellevelAnalysis, PixellevelPlots
```

Comment:
I think you forgot to add these files to git ^^, they are not in the PR.

Reply:
Indeed ^^. It's in now